charts/apisix/values.yaml

```diff
@@ -142,6 +142,7 @@ gateway:
     certCAFilename: ""
     http2:
       enabled: true
+    sslProtocols: "TLSv1.2 TLSv1.3"
```

Comment:
It is different from the previous value, do we need to consider compatibility? 🤔

Reply:
I think we need.

Reply:
Actually, I think we should drop the support of `TLSv1` and `TLSv1.1`, since both of them are unsafe and deprecated.

Reply:
> Actually, I think we should drop the support of `TLSv1` and `TLSv1.1`, since both of them are unsafe and deprecated.

I think it needs to be droped, but the behavior in the helm chart should be consistent with APISIX. Are these protocols droped in APISIX?

Reply:
OK, then we can drop it after APISIX changes its default value.

Reply:
See https://github.com/apache/apisix/blob/master/conf/config-default.yaml#L143.

Reply:
got it